packages/theme-toggler-extension/jupyterlab-theme-toggler/_version.py

```diff
@@ -0,0 +1,4 @@
+# This file is auto-generated by Hatchling. As such, do not:
+#   - modify
+#   - track in version control e.g. be sure to add to .gitignore
+__version__ = VERSION = '1.0.0'
```

Comment:
+1 for going with `1.0.0` for all these extensions :+1:

Reply:
And `_version.py` must be ignored as well. Will fix these soon.